ticket-msa/invoice/src/main/resources/project-defaults.yml

```diff
@@ -0,0 +1,10 @@
+swarm:
```

Comment:
Where/how is this file consumed?

Reply:
Is is Swarm/Thorntail config file - see https://docs.thorntail.io/2.2.0.Final/#default-thorntail-yaml-files_thorntail

Reply:
Ah, got it. What happened to good ol' _standalone.xml_. ;)